### PR TITLE
    Add import statements to bark skill

### PR DESCRIPTION
**Pull request by devvmh**
This makes the bark skill function on my mark2

Fixes https://github.com/jmillerv/mycroft2-skills/issues/8

### CODE REVIEW

**Comment by jmillerv** [comment]
I'll check this out on my machine and see what's going on Friday Night or Saturday day. I may have had some auto-import stuff going on that didn't get committed when I created this.

File: bark.mark2/__init__.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+    MycroftSkill,
+    intent_handler
+)
+
 class Bark(MycroftSkill):
     def __init__(self, skill_id: str):
         super().__init__(skill_id=skill_id, name="BarkSkill")
 
-    @intent_file_handler('bark.intent')
+    @intent_handler('bark.intent')
```
Comment (jmillerv) [inline bark.mark2/__init__.py:10]: Can you remove your pycachce files from this PR? I need to make a PR to update the gitignore and remove the pycache from this repo. We shouldn't be committing it. That was an oversight on my part.

File: bark.mark2/__init__.py [hunk 2]
```diff
@@ -1,8 +1,13 @@
+from mycroft.skills import (
```
Comment (jmillerv) [inline bark.mark2/__init__.py:1]: Yeah, the version on my machine matches this. I need to create a better workflow. Thanks for adding!

**Review by jmillerv — CHANGES_REQUESTED** [review]
Thanks for updating the repo, I added a comment about pycache.

**Comment by jmillerv** [comment]
Okay, tested locally. Yep.  Good call on the other suggestion for sym links so that what's on the machine matches the repo. 

When you remove the __pychache__ from the PR, I'll merge it.

**Comment by devvmh** [comment]
Whoops! I since added pycache to my global .gitignore so should be fine next time

**Review by jmillerv — APPROVED** [review]

**Pull request merged**